### DISCUSSION TITLE
Selecting multiple objects in pull-down list

### DISCUSSION THREAD

**Discussion by InFINNity-Deck** [mworion/MountWizzard4#302] [Feature Request] [answered]
Hi Michel,

last week I tried to program the following comets into my mount:

C/2022 E3 (ZTF)
C/2022 A2 (PanSTARSS)
C/2020 V2 (ZTF)

As you can see there is no easy way to select only these three when only using a part of text and a wildcard. The best I could do is using "C/20*", but that resulted in a long list of comets. I then used the Updater to remove the comets that I did not need one-by-one, a tedious job.

Would it be possible to either add objects to the mount (instead of replacing the current list) or to make a multi-select in the pull-down list (i.e. by Ctrl left mouse click)?

Maybe this is already possible, in which case I would love to hear how.

Thanks and kind regards,
Nicolàs

**Comment by mworion** [comment, marked as answer]
Hi Nicolas,
unfortunately adding data in the mount is not possible. They only accept a complete list.
But I could offer this:

https://www.dropbox.com/s/ylkm9ojy77pyc5d/Bildschirmaufnahme%202023-02-03%20um%2020.09.37.mov?dl=0

Michel

**Comment by InFINNity-Deck** [comment]
Hi Michel,

That would be a great solution!

⁣Thanks,
Nicolàs 

Op 3 feb. 2023 20:11, om 20:11, Michael ***@***.***> schreef:

**Comment by mworion** [comment]
Try to integrate it in the next release.

Von meinem iPad gesendet

**Comment by mworion** [comment]
Hi Nicolas,
if you install 3.0.1a1 you should be able to test the fix.
(Internal Updater, write 3.0.1a1 in available version text field and click start updater)
Michel

**Comment by InFINNity-Deck** [comment]
Hi Michel,

installed version 3.0.1a1 and it indeed works like a charm, thank you very much!
I did notice something strange while doing my first test:

![afbeelding](https://user-images.githubusercontent.com/66182061/218277607-fa0ec24c-d5eb-4a46-ba94-189f4588d484.png)

As you can see I selected four random comets, but only two arrived. Then I tried the three comets that have my current interest and that went fine:

![afbeelding](https://user-images.githubusercontent.com/66182061/218277666-30fc770b-3446-4db5-95b2-26c345318343.png)

Not sure if the comets in the first try had an issue or that it was a hick-up in the routine...
Anyway, I am happy with this mod, thank you very much!

Nicolàs
